docker/Dockerfile-p2

```diff
@@ -1,3 +1,35 @@
+#
+#  Copyright (c) 2018, 2023, Oracle and/or its affiliates. All rights reserved.

```

Comment:
I guess that would rather be 2019 instead of 2018, according to the file history.

Reply:
Ah yes correct